src/httpserver/handlers/ok.clj

```diff
@@ -0,0 +1,12 @@
+(ns httpserver.handlers.ok)
+
+(import '(com.td.HttpServer HttpResponse))
+(import '(com.td.HttpServer IHandler))
+
+(deftype ok-handler []
+  com.td.HttpServer.IHandler
+  (generateResponse [this]
+    (doto (HttpResponse.))))
```

Comment:
Huh? I'm confused. First of all, `doto` takes a data structure and a variable number of functions to run with it in the first argument spot. Check out the [documentation](https://clojuredocs.org/clojure.core/doto). What you have above is no different from simply `(HttpResponse.)`.

Second, uh, how does this work? Does `(HttpResponse.)` return a Response with 200 pre-set as the status?